### PR TITLE
feat: config for aws stacks TDE-173

### PR DESCRIPTION
**Pull request by amfage**
Like a phoenix from the ashes of my previous two pull requests...
This is to give the stack some more descriptive names, create an S3 bucket, and assign permissions.

### CODE REVIEW

File: infra/src/batch.ts [hunk 1]
```diff
@@ -39,13 +40,14 @@ export class AwsBatchStack extends Stack {
 
     instanceRole.addToPrincipalPolicy(new PolicyStatement({ resources: ['*'], actions: ['sts:AssumeRole'] }));
 
-    const bucket = new Bucket(this, 'Bucket', {
+    const topoProcessorBucket = new Bucket(this, 'TempBucket', {
```
Comment (paulfouquet) [inline infra/src/batch.ts:43]: Why not giving the same name for the variable (`const tempBucket = ...`)?

File: infra/src/submit.ts [hunk 2]
```diff
@@ -13,20 +13,18 @@ async function main(): Promise<void> {
   const environment = [
     { name: 'LINZ_CORRELATION_ID', value: correlationId },
     { name: 'LINZ_SSM_BUCKET_CONFIG_NAME', value: 'BucketConfig' },
-    { name: 'LDS_CACHE_BUCKET', value: 'linz-lds-cache' },
+    { name: 'LINZ_CACHE_BUCKET', value: 'linz-lds-cache' },
```
Comment (paulfouquet) [inline infra/src/submit.ts:16]: Did I miss something and `LDS_CACHE` is now named `LINZ_CACHE`?

**Review by paulfouquet — APPROVED** [review]
Looks good. Just a couple of question for my curiosity :)

File: infra/src/submit.ts (hunk 2 above)
Comment (amfage) [reply infra/src/submit.ts]: Oops yes that is from another PR, I've reverted as that PR is not yet merged.

Comment (blacha) [reply infra/src/submit.ts]: Ahh I renamed it so everything started with `LINZ_` no real reason just so it looks nice :D

**Review by MDavidson17 — APPROVED** [review]

**Pull request merged**